hq/app/utils/attempt/PlayIntegration.scala

```diff
@@ -10,7 +10,7 @@ object PlayIntegration extends Results {
   def attempt[A](action: => Attempt[Result])(implicit ec: ExecutionContext, assetsFinder: AssetsFinder): Future[Result] = {
     action.fold(
       { failures =>
-        Logger.error(failures.logMessage, failures.firstException.orNull)
+        Logger("attempt").error(failures.logMessage, failures.firstException.orNull)
```

Comment:
I think this would be a `PlayIntegration` error, rather than an `attempt` one (as with my other comments)